File: testprojects/csharp/benchmark/domain/lastfm/Album.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CSharpBenchmark.domain.lastfm
{

    public class Album
    {
        private string name;
        private DateTime releaseDate;

        public string getName()
        {
            return name;
        }

        public void setName(string name)
        {
            this.name = name;
        }

        public DateTime getReleaseDate()
        {
            return releaseDate;
        }

        public void setReleaseDate(DateTime releaseDate)
        {
            this.releaseDate = releaseDate;
        }
    }
}

File: testresources/java/benchmark_2012/domain/lastfm/Album.java
package domain.lastfm;

import java.util.Date;

public class Album {
	private String name;
	private Date releaseDate;
	
	public String getName() {
		return name;
	}
	
	public void setName(String name) {
		this.name = name;
	}
	
	public Date getReleaseDate() {
		return releaseDate;
	}
	
	public void setReleaseDate(Date releaseDate) {
		this.releaseDate = releaseDate;
	}
}